scriptmodules/ports/cannonball.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+# 
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+# 
+# See the LICENSE.md file at the top-level directory of this distribution and 
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="cannonball"
+rp_module_desc="cannonball - An Enhanced OutRun Engine"
+rp_module_menus="4+"
+rp_module_flags=""
+
+function depends_cannonball() {
+    getDepends libsdl2-dev libsdl2-image-dev libsdl2-ttf-dev libboost-dev
+}
+
+function sources_cannonball() {
+    gitPullOrClone "$md_build" https://github.com/VanFanel/cannonball.git
+}
+
+function build_cannonball() {
+    mkdir build
+    cd build
+    cmake -DTARGET=sdl2_rpi -DCMAKE_INSTALL_PREFIX:PATH="$md_inst" ../cmake
```

Comment:
also - if the module is rpi only other platforms need to be excluded in the flags like x11/mali but it should be possible to x86/x11 with this.